isis/src/base/objs/ImageExporter/ImageExporter.cpp

```diff
@@ -97,15 +98,23 @@ namespace Isis {
    * @param outputName The filename of the output cube
    * @param quality The quality of the output from 0 to 100, defaults to 100
    * @param compression The compression algorithm used. Image format specific.
+   * @param ui The optional user interface to set the input image pixel range.
    */
   void ImageExporter::write(FileName outputName, int quality,
-                            QString compression) {
+                            QString compression, UserInterface *ui) {
     ProcessExport &p = process();
-    if (!p.HasInputRange()) p.SetInputRange();
```

Comment:
Process::SetInputRange calls Application which caused seg-faults when running isis2std outside an application. Thankfully, this just calls Application::getUserInterface and then passes it to another function. so we have to add the new optional UserInterface argument and pass it on down through the different children.